HTML/syntax_test_html.html

```diff
@@ -234,6 +234,12 @@
         ##                        ^ - meta.attribute-with-value
         ##                         ^^ punctuation.definition.tag.end.html
 
+        <img srs="https://foo%2Fbar.com" />
```

Comment:
This should be `src`, right?
 
```suggestion
        <img src="https://foo%2Fbar.com" />
```

Reply:
Nope, it should ensure not to match `%2f` in something else that `href` or `src` tag. Can choose a different one if that counts.

Reply:
The positive test is one above.

Reply:
I've renamed the tag and added some comments to clarify the meaning of the test case.

Reply:
Ah, yes, thanks for clarifying 😄